tests/test_sg_trigger.py

```diff
@@ -51,6 +51,7 @@ def mocked_requests_post(*args, **kwargs):
     response = requests.Response()
     response.url = args[0]
     response._contents = kwargs
+    response.status_code = 200
```

Comment:
This was needed in Python 3, since when the event called `response.raise_for_status()` it would compare `None` to an `int` which is valid in Python 2, but not 3.